test/e2e/commands/test_ingest_xmlupload.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+import pytest
+
+from dsp_tools.commands.ingest_xmlupload.upload_xml import ingest_xmlupload
+from dsp_tools.models.exceptions import InputError
+
+
+@pytest.fixture()
+def base_path() -> Path:
+    """Get the root directory of the repository."""
+    return Path(__file__).parents[3]
```

Comment:
this looks hacky... but as long as it works... :)

Reply:
The problem I had with the solution that I discuses with Johannes (shutil.copy) was, that it actually moved the file to the root directory. Meaning that even after the test session the csv was no longer in the folder it actually needs to be. (It took me the longest time to figure out, why I kept getting the wrong error messages) I could not find how I can prevent that with shutil so I went with this.

Any other ideas?

Reply:
I really don't like this hack, but I'm optimistic that we can find a better way. 

You said that https://docs.python.org/3.10/library/shutil.html#shutil.copy moved the file instead of copying it? I cannot believe that... Perhaps you deleted the wrong file after the `yield`?

Reply:
Yup, and it works:

```
@pytest.fixture()
def _retrieve_mapping_file() -> None:
    """Put the mapping file into the cwd."""
    mapping_file = Path("testdata/dsp-ingest-data/mapping-00A2.csv")
    shutil.copy(mapping_file, ".")
    yield
    Path(mapping_file.name).unlink()


@pytest.mark.usefixtures("_retrieve_mapping_file")
def test_ingest_xmlupload() -> None:
    print("test function that does nothing")
```

Reply:
thanks

Reply:
> copying

The problem was I think that I didn't delete a file at all. What happened is that it disappeared from the original folder and ended up in the root folder...